### ISSUE TITLE
Which distributed strategy should I use when running training without GPU?

### ISSUE THREAD

**Issue by pingsutw** [tensorflow/tensorflow#36081]
There are many distributed strategy in Tensorflow,

- MirroredStrategy
- TPUStrategy
- MultiWorkerMirroredStrategy
- CentralStorageStrategy
- ParameterServerStrategy
- OneDeviceStrategy

Some of them run on one machine and broadcast model to different GPUs, and some of them use different GPUs on different machines.

My question if I don't have GPU in my server, which strategy should I use, so I can get the benefit when running distributed training?

I try to run MirroredStrategy, ParameterServerStrategy on four machines, but it seems that it's slower than running on a single machine.

**Comment by ymodak** [comment]
Sorry for the delay. Since you want to train on multiple machines then you should look at [`MultiWorkerMirroredStrategy`](https://www.tensorflow.org/guide/distributed_training#multiworkermirroredstrategy) or [`ParameterServerStrategy`](https://www.tensorflow.org/guide/distributed_training#parameterserverstrategy). 
Thanks!

**Comment by pingsutw** [comment]
I will give it a try

**Comment by nikitamaia** [comment]
Scaling vertically might lead to better results compared to using multiple machines if you don't have GPUs. The overhead of synchronizing the gradients across your machines with CPUs could explain why scaling to four machines is not reducing your training time.

**Comment by nikitamaia** [comment]
As mentioned by @ymodak, both MultiWorkerMirroredStrategy or ParameterServerStrategy support training across multiple machines. Closing this issue now since it is not a bug, but feel free to create a new issue if you run into performance issues with these strategies in the future.

**Comment by pingsutw** [comment]
@nikitamaia Thanks for the reply and help
